influxdb-line-protocol.js

```diff
@@ -62,7 +62,7 @@ function joinObject(obj, withFormatting, config) {
   return Object.keys(obj)
     .map(key => {
       let override = config.typeMappings.find( (i) => i.fieldName == key);
-      let numType = override?.fieldType || config.defaultTypeMapping;
+      let numType = override.fieldType || config.defaultTypeMapping;
```

Comment:
This changes semantics though. If override is not found, `override.fieldType` will cause an error. So `(override && override.fieldType) || config.defaultTypeMapping` might be better.

Reply:
ok, sorry, I'm not a specialist coder (I only knew "?" as the ternary operator, not as optional chaining operator).  I suppose my node.js version is too old and that caused it to fail loading this node?  So I kinda removed it without further thinking about it.  So your suggestion seems ok to me.